Lib/test/clinic.test.c

```diff
@@ -4198,44 +4198,6 @@ static PyObject *
 test_vararg_and_posonly_impl(PyObject *module, PyObject *a, PyObject *args)
 /*[clinic end generated code: output=0c11c475e240869e input=2c49a482f68545c0]*/
 
-PyDoc_STRVAR(test_vararg_and_posonly__doc__,
-"test_vararg_and_posonly($module, a, /, *args)\n"
-"--\n"
-"\n");
-
-#define TEST_VARARG_AND_POSONLY_METHODDEF    \
-    {"test_vararg_and_posonly", _PyCFunction_CAST(test_vararg_and_posonly), METH_FASTCALL, test_vararg_and_posonly__doc__},
-
-static PyObject *
-test_vararg_and_posonly_impl(PyObject *module, PyObject *a, Py_ssize_t nargs,
```

Comment:
@skirpichev You added this one in 8c22eba877225ab29cd64672dcb97f09a5f7336f but it already existed above. Can you explain what you wanted to test here please?

Reply:
Good catch, but both functions existed before.

Reply:
Yes but maybe you knew why they were duplicated ...